File: backend/BookstoreAPI/Models/Articulo.cs
namespace BookstoreAPI.Models
{
    public class Articulo
    {
        public int Id { get; set; }
        public string? Codigo { get; set; }
        public string? Descripcion { get; set; }
        public string? CodBarras { get; set; }
        public string? Observaciones { get; set; }
        public int? Tomos { get; set; }
        public string? Tema { get; set; }
        public decimal? Precio { get; set; }
    }
}

File: frontend/bookstore-app/src/types/articulo.ts
export interface Articulo {
  id: number;
  codigo?: string;
  descripcion?: string;
  codBarras?: string;
  observaciones?: string;
  tomos?: number;
  tema?: string;
  precio?: number;
}

export interface CreateArticuloDto {
  codigo?: string;
  descripcion: string;
  codBarras?: string;
  observaciones?: string;
  tomos?: number;
  tema?: string;
  precio?: number;
}

export interface UpdateArticuloDto {
  codigo?: string;
  descripcion: string;
  codBarras?: string;
  observaciones?: string;
  tomos?: number;
  tema?: string;
  precio?: number;
}
